docs/examples/frameworks/closed_single_award.json

```diff
@@ -0,0 +1,116 @@
+{
+  "uri": "https://example.com/ocds/ocds-r6ebe6-95f19d21-d066-4931-a431-071211033960/1",
+  "version": "1.1",
+  "publishedDate": "2019-03-01T16:30:52Z",
+  "publisher": {
+    "name": "Open Contracting Partnership"
+  },
+  "license": "https://creativecommons.org/publicdomain/zero/1.0/",
+  "publicationPolicy": "http://example.com/publicationPolicy",
+  "extensions": [
+    "https://raw.githubusercontent.com/open-contracting-extensions/ocds_competitive_extension/master/extension.json"
+  ],
+  "releases": [
+    {
+      "ocid": "ocds-r6ebe6-95f19d21-d066-4931-a431-071211033960",
+      "id": "1",
+      "date": "2020-10-01T00:00:00Z",
+      "tag": [
+        "award",
+        "contract"
+      ],
+      "initiationType": "tender",
+      "buyer": {
+        "name": "The Common Services Agency (more commonly known as NHS National Services Scotland) (‘NSS’)",
+        "id": "GB-NHS-YA7"
+      },
+      "parties": [
+        {
+          "name": "The Common Services Agency (more commonly known as NHS National Services Scotland) (‘NSS’)",
+          "id": "GB-NHS-YA7",
+          "identifier": {
+            "scheme": "GB-NHS",
+            "id": "YA7"
+          },
+          "roles": [
+            "buyer",
+            "procuringEntity"
+          ]
+        },
+        {
+          "name": "Movianto UK",
+          "id": "1",
+          "roles": [
+            "supplier"
+          ]
+        }
+      ],
+      "tender": {
+        "id": "1",
+        "procuringEntity": {
+          "name": "The Common Services Agency (more commonly known as NHS National Services Scotland) (‘NSS’)",
+          "id": "GB-NHS-YA7",
+          "competitive": false
```

Comment:
this should be outside the procuringEntity block

Reply:
Fixed in https://github.com/open-contracting/standard/pull/1123/commits/71ff34beadcf46d1c4ab5941b9a740a61b15a990